inscriptions/ISic003377.xml

```diff
@@ -184,13 +184,15 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" xml:lang="la" resp="#VM">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
+                <ab>
+                    <lb n="1"/><supplied reason="lost">χ</supplied>αῖρε
+                </ab>
             </div>
             <div type="apparatus">
                 <listApp>
                     <app>
-                        <note>Text after </note>
+                        <note>Text after photograph</note>
```

Comment:
this should have a similar note in the commentary to the previous one, as currently unidentified and possibly unpublished [although I have a nagging feeling that I *have* at some point seen this piece in NSA; and there is a German post-doc who was working on this sort of material in the Orsi a few years back who alsmost certainly knows more about it, and I must follow up]